client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -78,6 +78,16 @@ const siteVertical: Reducer< SiteVertical | undefined, OnboardAction > = ( state
 	return state;
 };
 
+const wasVerticalSkipped: Reducer< boolean, OnboardAction > = ( state = false, action ) => {
+	if ( action.type === 'SKIP_SITE_VERTICAL' ) {
+		return true;
+	}
+	if ( action.type === 'RESET_ONBOARD_STORE' ) {
+		return false;
+	}
```

Comment:
This works, although I'm a bit skeptical of storing this in global state…

I guess we either need to track progress explicitly in state (we've been avoiding) or include things like this so we can infer progress based on what we do have in state.

---

Should setting a vertical also set this to false?

Reply:
Tracking progress will trade some of the _modularity_ we currently have 🤔 Until now we had something like this only set indirectly around `siteVertical` being defined of not.

> Should setting a vertical also set this to false?

I thought skipping vertical selection would be an action we store during the entire user flow (until site creation). That's why I included this in description:
> If you do this, your option will be persisted in local storage so next time you visit Gutenboarding both inputs should be displayed even if vertical field is empty.

@dubielzyk could you confirm this is correct? ☝️

Reply:
But you can come back, set a vertical and delete it. The store still shows like you skipped, but did you? 🤔

Reply:
> But you can come back, set a vertical and delete it.

We remember that you skipped this step recently (without finishing Gutenboarding site creation). After all you explicitly pressed skip link once (except if someone else was using your browser). 

So if you deleted the vertical (accidentally or not) you still have the Site Name input and Next CTA button:
<img width="700" alt="Screenshot 2020-04-30 at 10 33 48" src="https://user-images.githubusercontent.com/14192054/80684554-85265780-8ace-11ea-8196-40e8c408d821.png">

Instead of having to press once again on the skip button:
<img width="700" alt="Screenshot 2020-04-30 at 10 33 57" src="https://user-images.githubusercontent.com/14192054/80684456-61fba800-8ace-11ea-929e-33e9ac44005b.png">